### PR TITLE
Prepare V2 -> V1 Diffs

### PR DESCRIPTION
**Pull request by ryancdickson**
This is a staging ground for updates in response to SC80 V1, preparing for SC80 V2.

Links:
- [V1 Discussion](https://archive.cabforum.org/pipermail/servercert-wg/2024-September/004825.html)

### CODE REVIEW

File: docs/BR.md [hunk 1]
```diff
@@ -751,7 +751,14 @@ The Random Value SHALL remain valid for use in a confirming response for no more
...
+Effective January 15, 2025, when issuing Subscriber Certificates…
+- The CA MUST NOT rely on Domain Contact information obtained using an HTTPS website, regardless of whether previously obtained information is within the allowed reuse period.
+- The CA MUST NOT rely on Domain Contact information obtained using the WHOIS protocol (RFC 3912) or the Registry Data Access Protocol (RFC 7482) if the requested Domain Name contains a ccTLD, regardless of whether previously obtained information is within the allowed reuse period.
+- When obtaining Domain Contact information using the WHOIS protocol, the CA MUST query IANA's WHOIS server and follow referrals to the appropriate gTLD WHOIS server.
+- When obtaining Domain Contact information using the Registry Data Access Protocol, the CA MUST utilize IANA's bootstrap file to identify and query the correct RDAP server for the domain.
+- The CA SHOULD NOT rely on cached 1) WHOIS server information or 2) RDAP bootstrap data from IANA to ensure that it relies upon up-to-date and accurate information.
```
Comment (AGWA) [inline docs/BR.md]: I propose:

> The CA MUST NOT rely on cached 1) WHOIS server information or 2) RDAP bootstrap data from IANA that is more than 48 hours old,  to ensure that it relies upon up-to-date and accurate information

Rationale: 48 hours is the TTL on the NS records in the root zones, so this provides parity with DNS.  I don't think CAs should have discretion to ignore this.

(Ditto for the other sections where this language is used.)

Comment (ryancdickson) [reply docs/BR.md]: Thanks for your review and feedback, Andrew! Your suggestion and justification are reasonable to me.

I'll leave this comment open to help facilitate additional discussion.

File: docs/BR.md [hunk 2]
```diff
@@ -751,7 +751,14 @@ The Random Value SHALL remain valid for use in a confirming response for no more
...
+
+Effective September 15, 2025, the CA MUST NOT rely on this method.
```
Comment (wthayer) [inline docs/BR.md]: We should also specify what happens with data reuse, i.e. copy this from 3.2.2.4.3: "Prior validations using this method and validation data gathered according to this method SHALL NOT be used to issue certificates."

Comment (ryancdickson) [reply docs/BR.md]: @wthayer - thanks for your comment. To confirm my understanding, the proposal would be to similar to the following?

**From:** _Effective September 15, 2025, the CA MUST NOT rely on this method._

**To:** _Effective September 15, 2025:_
_- the CA MUST NOT rely on this method._
_- Prior validations using this method and validation data gathered according to this method MUST NOT be used to issue certificates._

[Pedantic] Is there well-accepted convention within the Forum re: "MUST" vs "SHALL"? Generally speaking, I'm on Team "[MUST](https://www.plainlanguage.gov/guidelines/conversational/shall-and-must/#:~:text=Use%20%E2%80%9Cmust%E2%80%9D%20not%20%E2%80%9Cshall,express%20a%20requirement%20or%20obligation.)."

Comment (wthayer) [reply docs/BR.md]: Yes, that's my suggestion, thank you.

I'm not an expert in IETF-speak, but I think it's fine to use MUST here. I simply copied the SHALL language from a different subsection.

File: docs/BR.md (hunk 1 above)
Comment (ryancdickson) [reply docs/BR.md]: Given no further comments, I actioned this comment in [b740449](https://github.com/ryancdickson/staging/commit/b740449d80c71ec4f3cefab2e68eb86ba2ccc924).

File: docs/BR.md (hunk 2 above)
Comment (ryancdickson) [reply docs/BR.md]: Given no further comments, I actioned this comment in [b740449](https://github.com/ryancdickson/staging/commit/b740449d80c71ec4f3cefab2e68eb86ba2ccc924).

**Comment by ryancdickson** [comment]
Given no response to discussion on [list](https://archive.cabforum.org/pipermail/servercert-wg/2024-September/004919.html), [b740449](https://github.com/ryancdickson/staging/commit/b740449d80c71ec4f3cefab2e68eb86ba2ccc924) also shifts the sunset date from 9/15/2025 to 7/15/2025.

**Comment by ryancdickson** [comment]
Closing, overcome by newer versions of the ballot.

**Pull request closed without merging**